armi/reactor/tests/test_reactors.py

```diff
@@ -272,6 +270,25 @@ def test_factorySortSetting(self):
         a1 = [a.name for a in r1.core]
         self.assertNotEqual(a0, a1)
 
+    def test_sortChildren(self):
```

Comment:
this test failed on main due to the `if issubclass(c.__class__, Composite):` issue